roles/cs.redis/tasks/configure.yml

```diff
@@ -111,8 +111,23 @@
     - "echo never > /sys/kernel/mm/transparent_hugepage/defrag"
   when: _redis_enabled_thd.changed
 
-- name: Ensure Redis service is started
+- name: Stop Redis/Valkey in case of migration
   service:
-    name: "{{ redis_daemon }}"
+    name: >-
+      {{ redis_daemon
+         | regex_replace('^redis', 'valkey')
+         | regex_replace('^valkey', 'redis') if redis_daemon.startswith('valkey') else
+         redis_daemon | regex_replace('^redis', 'valkey') }}
+    state: stopped
+    enabled: false
+  ignore_errors: yes
```

Comment:
why ignore_errors?
because it might not exist?
in that case better use would be to detect it with `service_facts` instead.